### ISSUE TITLE
the missing baseUrl config causes error

### ISSUE THREAD

**Issue by cdll** [openks/gulp-pre-link#2]
we see that in the gulp-pre-link versioned 2.2, `add new parameter baseUrl if there is no this parameter,the path is based on the same with gulpfile.js`, here below:

```
fs.js:549
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                                                  ^

Error: ENOENT: no such file or directory, open 'undefinedsrc/include/common-head-inapp.html'
    at Error (native)
    at Object.fs.openSync (fs.js:549:18)
    at Object.fs.readFileSync (fs.js:397:15)
```

in my html

``` html
  <link rel='import' href='src/include/common-head-inapp.html'>
```

the `common-head-inapp.html` file was in the folder `/src/include` that has the same baseUrl with my `/gulpfile.js` . could u pls help me with it~

**Comment by openks** [comment]
since the computer around don't have the dev environment,i can do nothing before monday, i'll deal with this on monday when i come back

**Comment by openks** [comment]
fixed on v2.3.0
